#,Node,Feature,TestCase,Result,ResultType,ExecutedOn,Description,Function,Module,Devices,KnownIssue
1,gw2,Regression,ft_add_unknownvlan_interface,Pass,Mapped,2024-01-04 01:10:04,Test case passed,test_ft_add_unknownvlan_interface,switching/test_vlan.py,"V1, V2",
2,gw2,Regression,ft_vlan_delete_with_member,Pass,Mapped,2024-01-04 01:10:19,Test case passed,test_ft_vlan_delete_with_member,switching/test_vlan.py,"V1, V2",
3,gw3,Regression,FtOpSoSwLagFn042,Pass,Mapped,2024-01-04 01:11:33,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
4,gw3,Regression,FtOpSoSwLagFn041,Pass,Mapped,2024-01-04 01:11:33,Test case passed,test_ft_portchannel_with_vlan_variations,switching/test_portchannel.py,"V3, V4",
5,gw2,Regression,ft_vlan_trunk_tagged,Pass,Mapped,2024-01-04 01:11:44,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
6,gw2,Regression,ft_show_vlan_brief,Pass,Mapped,2024-01-04 01:11:44,Test case passed,test_ft_vlan_trunk_tagged,switching/test_vlan.py,"V1, V2",
7,gw3,PortChannel Config Enhancements,FtOpSoSwPominFn007,Pass,Mapped,2024-01-04 01:11:57,Test case passed,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,switching/test_portchannel.py,"V3, V4",
8,gw3,Regression,ft_ssh_service_disable,Pass,Mapped,2024-01-04 01:18:32,Test case passed,test_ft_ssh_service_disable,system/test_ssh.py,"V3, V4",
9,gw2,warmboot,ft_reboot_fdb_fast_reboot,ScriptError,Mapped,2024-01-04 01:24:18,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
10,gw2,warmboot,ft_max_vlan_save_reload,ScriptError,Mapped,2024-01-04 01:24:18,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
11,gw2,warmboot,FtOpSoSysFRFn005,ScriptError,Mapped,2024-01-04 01:24:18,Exception TypeError 'NoneType' object is not subscriptable in file /apis/system/basic.py at line 1389,test_ft_vlan_save_config_warm_and_fast_reboot,switching/test_vlan.py,"V1, V2",
12,gw3,Regression,ft_ssh_add_user_verify,Fail,Mapped,2024-01-04 01:29:08,Test case Failed.,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
13,gw3,Control Plane ACL,test_ft_controlplane_acl_service_sshv4,Pass,SubTest,2024-01-04 01:29:08,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
14,gw3,Control Plane ACL,test_ft_controlplane_acl_service_sshv6,Pass,SubTest,2024-01-04 01:29:08,SSH failed,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
15,gw3,Control Plane ACL,test_ft_controlplane_acl_service_snmp,Fail,SubTest,2024-01-04 01:29:08,SNMP output is not found after walk of with control plane ACL service SNMP after reboot OID,test_ft_ssh_add_user_verify,system/test_ssh.py,"V3, V4",
16,gw3,Regression,CETA_SONIC_67575,Pass,Mapped,2024-01-04 01:29:31,Test case passed,test_ft_verify_logs_at_the_time_of_ssh_login_success,system/test_ssh.py,"V3, V4",
17,gw2,Regression,ft_lldp_LocManAddrLen,Fail,Mapped,2024-01-04 01:31:16,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrLen,system/test_lldp.py,"V1, V2",
18,gw2,Regression,ft_lldp_LocManAddrlfld,Fail,Mapped,2024-01-04 01:31:17,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrlfld,system/test_lldp.py,"V1, V2",
19,gw2,Regression,ft_lldp_LocManAddrEntry,Fail,Mapped,2024-01-04 01:31:17,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_LocManAddrEntry,system/test_lldp.py,"V1, V2",
20,gw2,Regression,ft_lldp_ConfigManAddrEntry,Fail,Mapped,2024-01-04 01:31:18,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_ConfigManAddrEntry,system/test_lldp.py,"V1, V2",
21,gw2,Regression,ft_lldp_lldplocportid,Fail,Mapped,2024-01-04 01:31:18,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocportid,system/test_lldp.py,"V1, V2",
22,gw2,Regression,ft_lldp_lldplocsysname,Fail,Mapped,2024-01-04 01:31:18,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysname,system/test_lldp.py,"V1, V2",
23,gw2,Regression,ft_lldp_lldplocsysdesc,Fail,Mapped,2024-01-04 01:31:19,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_lldplocsysdesc,system/test_lldp.py,"V1, V2",
24,gw2,SNMP Agent Optimization,ft_snmp_lldp_prem_manaddr_table,Fail,Mapped,2024-01-04 01:31:19,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_rem_man_addr_table,system/test_lldp.py,"V1, V2",
25,gw2,Regression,FtOpSoSwlldpFn011,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V1, V2",
26,gw2,Regression,FtOpSoSwlldpFn010,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V1, V2",
27,gw2,Regression,FtOpSoSwlldpFn009,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V1, V2",
28,gw2,Regression,FtOpSoSwlldpFn008,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V1, V2",
29,gw2,Regression,FtOpSoSwlldpFn004,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_non_default_config,system/test_lldp.py,"V1, V2",
30,gw2,Regression,FtOpSoSwlldpFn013,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_docker_restart,system/test_lldp.py,"V1, V2",
31,gw2,Regression,lldp_verify_counters,Fail,Mapped,2024-01-04 01:31:20,module configuration failed. LLDP neighbors information is not found after polling @5445,test_lldp_verify_counters,system/test_lldp.py,"V1, V2",
32,gw2,Regression,ft_lldp_with_remote_server,Fail,Mapped,2024-01-04 01:31:21,module configuration failed. LLDP neighbors information is not found after polling @5445,test_ft_lldp_with_remote_server,system/test_lldp.py,"V1, V2",
33,gw3,Regression,FtOpSoSyInIpFn002,DUTFail,Mapped,2024-01-04 01:37:12,Error response from daemon: Container is not running,test_ft_port_fn_verify_shut_noshut,system/test_interface.py,"V3, V4",
34,gw4,Instrumentation,ft_sf_wm_telemetry_interval,Pass,Mapped,2024-01-04 01:38:55,verification of configured telemetry interval and reset to default interval is successful,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V2,
35,gw4,Instrumentation Enhancements,ft_sf_snapshot_tam_Error_Logs,Fail,SubTest,2024-01-04 01:38:55,Error logs are getting generated on the device,test_ft_watermark_telemetry_interval,system/test_snapshot.py,V2,
36,gw6,Regression,ft_ntp_message_log_display_with_correct_time,ScriptError,Mapped,2024-01-04 01:43:26,Exception IndexError list index out of range in file /apis/system/ntp.py at line 347,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V1,
37,gw6,Regression,ft_ntp_disable_ntp_enable_ntp,ScriptError,Mapped,2024-01-04 01:43:26,Exception IndexError list index out of range in file /apis/system/ntp.py at line 347,test_ft_ntp_disable_enable_with_message_log,system/test_ntp.py,V1,
38,gw6,Regression,ft_ntp_existsconfig,Pass,Mapped,2024-01-04 01:43:40,Test case passed,test_ntp_exists_config,system/test_ntp.py,V1,
39,gw4,Regression,ft_tacacsauth_enable_pamfile_created,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
40,gw4,Regression,ft_tacacs_ssh_login,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
41,gw4,Regression,ft_tacacs_single_server_for_auth,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
42,gw4,Regression,ft_tacacs_default_login,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
43,gw4,Regression,executing_server_key_in_appropriate_mode,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
44,gw4,Regression,auth_shared_keys ,Fail,Mapped,2024-01-04 01:44:55,PAM file is not created,test_ft_tacacs_ssh_login_with_tacacs_operations,security/test_tacacs.py,V2,
45,gw5,Regression,ft_snmp_sysName,Fail,Mapped,2024-01-04 01:46:52,sysname output is not in sync between system and snmp,test_ft_snmp_sysName,system/test_snmp.py,V4,
46,gw5,Regression,ft_snmp_test_syUpTime,Pass,Mapped,2024-01-04 01:46:58,Test case passed,test_ft_snmp_test_syUpTime,system/test_snmp.py,V4,
47,gw5,Regression,ft_snmp_sysDescr,Pass,Mapped,2024-01-04 01:47:06,Test case passed,test_ft_snmp_sysDescr,system/test_snmp.py,V4,
48,gw5,Regression,FtOpSoSySnFn008,Pass,Mapped,2024-01-04 01:47:13,Test case passed,test_ft_snmp_mib_2,system/test_snmp.py,V4,
49,gw5,Regression,FtOpSoSySnFn009,Pass,Mapped,2024-01-04 01:47:18,Test case passed,test_ft_snmp_if_mib_all,system/test_snmp.py,V4,
50,gw5,Regression,FtOpSoSySnFn010,Pass,Mapped,2024-01-04 01:47:22,Test case passed,test_ft_snmp_entity_mib_all,system/test_snmp.py,V4,
51,gw5,Regression,FtOpSoSySnFn013,Pass,Mapped,2024-01-04 01:47:27,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
52,gw5,Regression,FtOpSoSySnFn012,Pass,Mapped,2024-01-04 01:47:27,Test case passed,test_ft_snmp_dot1q_dot1db_mib,system/test_snmp.py,V4,
53,gw5,Regression,FtOpSoSySnFn015,Pass,Mapped,2024-01-04 01:47:36,Test case passed,test_ft_snmp_root_node_walk,system/test_snmp.py,V4,
54,gw5,Regression,FtOpSoSySnFn016,Pass,Mapped,2024-01-04 01:47:46,Test case passed,test_ft_snmp_ipAddressRowStatus_ipv6,system/test_snmp.py,V4,
55,gw5,Regression,FtOpSoSySnFn017,Pass,Mapped,2024-01-04 01:47:51,Test case passed,test_ft_snmp_ipAddressStorageType_ipv6,system/test_snmp.py,V4,
56,gw5,Regression,FtOpSoSySnFn020,Pass,Mapped,2024-01-04 01:47:55,Test case passed,test_ft_snmp_ipv6_If_Forward_default_HopLimit,system/test_snmp.py,V4,
57,gw5,Regression,FtOpSoSySnFn021,Pass,Mapped,2024-01-04 01:48:00,Test case passed,test_ft_snmp_ipv6scope_index_table,system/test_snmp.py,V4,
58,gw5,Regression,test_snmp_ip_forward_table,Pass,Mapped,2024-01-04 01:48:08,Test case passed,test_ft_snmp_ip_forward_table,system/test_snmp.py,V4,
59,gw5,SNMP Agent Optimization,ft_snmp_ifXTable,Pass,Mapped,2024-01-04 01:48:14,Test case passed,test_ft_snmp_ifx_table,system/test_snmp.py,V4,
60,gw5,SNMP Agent Optimization,ft_snmp_ip_System_Stats_Table,Pass,Mapped,2024-01-04 01:48:18,Test case passed,test_ft_snmp_ip_System_Stats_Table,system/test_snmp.py,V4,
61,gw5,SNMP Agent Optimization,ft_snmp_ip_IfStats_Table,Pass,Mapped,2024-01-04 01:48:24,Test case passed,test_ft_snmp_ip_IfStats_Table,system/test_snmp.py,V4,
62,gw5,SNMP Agent Optimization,ft_snmp_ip_Address_Table,Pass,Mapped,2024-01-04 01:48:27,Test case passed,test_ft_snmp_ip_Address_Table,system/test_snmp.py,V4,
63,gw5,SNMP Agent Optimization,ft_snmp_ip_NetToPhysical_Table,Pass,Mapped,2024-01-04 01:48:32,Test case passed,test_ft_snmp_ip_NetToPhysical_Table,system/test_snmp.py,V4,
64,gw5,SNMP Agent Optimization,ft_snmp_icmp_Msgs,Pass,Mapped,2024-01-04 01:48:37,Test case passed,test_ft_snmp_icmp_Msgs,system/test_snmp.py,V4,
65,gw5,SNMP Agent Optimization,ft_snmp_tcp_mib,Pass,Mapped,2024-01-04 01:48:44,Test case passed,test_ft_snmp_tcp_mib,system/test_snmp.py,V4,
66,gw5,SNMP Agent Optimization,ft_snmp_udp_mib,Pass,Mapped,2024-01-04 01:48:49,Test case passed,test_ft_snmp_udp_mib,system/test_snmp.py,V4,
67,gw5,SNMP Agent Optimization,ft_snmp_snmpv2_mib,Pass,Mapped,2024-01-04 01:48:54,Test case passed,test_ft_snmp_snmpv2_mib,system/test_snmp.py,V4,
68,gw5,SNMP Agent Optimization,ft_snmp_host_resource_mib,Pass,Mapped,2024-01-04 01:48:59,Test case passed,test_ft_snmp_host_resource_mib,system/test_snmp.py,V4,
69,gw5,SNMP Agent Optimization,ft_snmp_framework_mib,Pass,Mapped,2024-01-04 01:49:04,Test case passed,test_ft_snmp_framework_mib,system/test_snmp.py,V4,
70,gw5,SNMP Agent Optimization,ft_snmp_mpd_mib,Pass,Mapped,2024-01-04 01:49:09,Test case passed,test_ft_snmp_mpd_mib,system/test_snmp.py,V4,
71,gw5,SNMP Agent Optimization,ft_snmp_target_mib,Pass,Mapped,2024-01-04 01:49:14,Test case passed,test_ft_snmp_target_mib,system/test_snmp.py,V4,
72,gw5,SNMP Agent Optimization,ft_snmp_notification_mib,Pass,Mapped,2024-01-04 01:49:19,Test case passed,test_ft_snmp_notification_mib,system/test_snmp.py,V4,
73,gw5,SNMP Agent Optimization,ft_snmp_user_based_sm_mib,Pass,Mapped,2024-01-04 01:49:24,Test case passed,test_ft_snmp_user_based_sm_mib,system/test_snmp.py,V4,
74,gw5,SNMP Agent Optimization,ft_snmp_view_based_acm_mib,Pass,Mapped,2024-01-04 01:49:29,Test case passed,test_ft_snmp_view_based_acm_mib,system/test_snmp.py,V4,
75,gw5,SNMP Agent Optimization,ft_snmp_ent_physical_table,Pass,Mapped,2024-01-04 01:49:33,Test case passed,test_ft_snmp_ent_physical_table,system/test_snmp.py,V4,
76,gw5,SNMP Agent Optimization,ft_snmp_dot3_stats_table,Pass,Mapped,2024-01-04 01:49:38,Test case passed,test_ft_snmp_dot3_stats_table,system/test_snmp.py,V4,
77,gw5,SNMP Agent Optimization,ft_net_snmp_agent_mib,Pass,Mapped,2024-01-04 01:49:44,Test case passed,test_ft_net_snmp_agent_mib,system/test_snmp.py,V4,
78,gw5,SNMP Agent Optimization,ft_net_snmp_vacm_mib,Pass,Mapped,2024-01-04 01:49:49,Test case passed,test_ft_net_snmp_vacm_mib,system/test_snmp.py,V4,
79,gw5,SNMP Agent Optimization,ft_snmp_ucd_diskio_mib,Pass,Mapped,2024-01-04 01:49:54,Test case passed,test_ft_snmp_ucd_diskio_mib,system/test_snmp.py,V4,
80,gw5,SNMP Agent Optimization,ft_snmp_ucd_la_table,Pass,Mapped,2024-01-04 01:49:58,Test case passed,test_ft_snmp_ucd_la_table,system/test_snmp.py,V4,
81,gw5,SNMP Agent Optimization,ft_snmp_ucd_system_stats,Pass,Mapped,2024-01-04 01:50:03,Test case passed,test_ft_snmp_ucd_system_stats,system/test_snmp.py,V4,
82,gw5,SNMP MIB and Traps,SNMPQBr003,Fail,Mapped,2024-01-04 01:50:10,SNMP output is not found after walk of dot1qTpFdbPort OID,test_ft_snmp_dot1q_tp_fdb_port,system/test_snmp.py,V4,
83,gw5,SNMP MIB and Traps,SNMPQBr002,Pass,Mapped,2024-01-04 01:50:15,Test case passed,test_ft_snmp_dot1q_tp_fdb_address,system/test_snmp.py,V4,
84,gw5,In-Memory Debug Log,FtOpSoSysImLogFn012,EnvFail,Mapped,2024-01-04 01:50:34,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
85,gw5,SNMP MIB and Traps,SNMPTrap004,EnvFail,Mapped,2024-01-04 01:50:34,Test case Failed Couldn't get the snmptrapd process details from snmp trap server. Please check the snmp trap server process in the server.,test_ft_snmp_coldstart_trap,system/test_snmp.py,V4,
86,gw5,Regression,FtOpSoSySnFn023,Fail,Mapped,2024-01-04 01:50:53,sysname output is not in sync between system and snmp after docker restart,test_ft_snmp_docker_restart,system/test_snmp.py,V4,
